test/TestBase.php

```diff
@@ -64,33 +65,63 @@ final protected function createOptionsFromArgv(array $argv, ?string $cwd = null)
         return Options::fromConsoleInput($input, $cwd ?? __DIR__);
     }
 
+    public const SANITIZE = [
+        '/^Time: .+/m',
+        '/^Random Seed: .+/m',
+        '/^.+[(]100%[)]$/m',
+        '/Warming cache for static analysis ... \[[^]]+\]/',
+        "/flowId='\d+'/",
+        "/duration='\d+'/",
+    ];
+
     final protected function runRunner(): RunnerResult
     {
-        $output      = new BufferedOutput();
-        $runnerClass = $this->runnerClass;
-
-        $options                              = $this->createOptionsFromArgv($this->bareOptions);
-        $shouldPutEnvForParatestTestingItSelf = $options->noTestTokens;
-        $runner                               = new $runnerClass($options, $output);
-        if ($shouldPutEnvForParatestTestingItSelf) {
-            $prevToken       = getenv(Options::ENV_KEY_TOKEN);
-            $prevUniqueToken = getenv(Options::ENV_KEY_UNIQUE_TOKEN);
-
-            putenv(Options::ENV_KEY_TOKEN);
-            putenv(Options::ENV_KEY_UNIQUE_TOKEN);
-            unset($_SERVER[Options::ENV_KEY_TOKEN]);
-            unset($_SERVER[Options::ENV_KEY_UNIQUE_TOKEN]);
-        }
-
-        $exitCode = $runner->run();
-        if ($shouldPutEnvForParatestTestingItSelf) {
-            putenv(Options::ENV_KEY_TOKEN . '=' . $prevToken);
-            putenv(Options::ENV_KEY_UNIQUE_TOKEN . '=' . $prevUniqueToken);
-            $_SERVER[Options::ENV_KEY_TOKEN]        = $prevToken;
-            $_SERVER[Options::ENV_KEY_UNIQUE_TOKEN] = $prevUniqueToken;
+        $last = null;
```

Comment:
Please restore the original code here, and add a new dedicated test for the `--functional` option, with both numeric data names and string data names